demo/config.js

```diff
@@ -588,13 +588,27 @@ shakaDemo.Config = class {
         .addNumberInput_('Max playback rate',
             'streaming.liveSync.maxPlaybackRate',
             /* canBeDecimal= */ true,
-            /* canBeZero= */ false)
+            /* canBeZero= */ true)
```

Comment:
Do we really want this? Because if it is zero, it implies that there is no movement in the video.

Reply:
it's not strictly necessary, but if you wanted to set it to a value less than 1, it'll show a warning. Probably should be fixed in the check for zero instead.

Reply:
Maybe yes, and in another PR. Thanks!

Reply:
I double-checked, and apparently the issue isn't in the config code, but with the pattern for showing the warning. Seems like it doesn't properly account for <1 values.

Reply:
PRed the change here: https://github.com/shaka-project/shaka-player/pull/6903